bin/node/runtime/src/lib.rs

```diff
@@ -471,6 +471,8 @@ parameter_types! {
 }
 
 impl pallet_staking::Config for Runtime {
+	const MAX_NOMINATIONS: u32 =
+		<NposCompactSolution16 as sp_npos_elections::CompactSolution>::LIMIT as u32;
```

Comment:
name is inconsistent with below
```suggestion
		<NposSolution16 as sp_npos_elections::CompactSolution>::LIMIT as u32;
```

Reply:
All renamed now to `NposCompactSolution16 `

Reply:
becaues they have `#[compact]`

Reply:
I guess in your local branch :-)